examples/prisma.md

```diff
@@ -0,0 +1,116 @@
+# Prisma
+
+There are two ways to use Prisma in your Cloudflare Workers:
+
+- Enable Accelerate
+- Enable Driver Adapters
+
+### Start Prisma
+
+First, start Prisma on your Hono Cloudflare Workers. Here, I am using neon.tech as my PostgreSQL database, but you can use whichever database suits your project.
+
```

Comment:
- Change "start" to "install".
- For neon say `I am using the [neon.tech](https://neon.tech/) free tier`...